lib/src/base-styles/_nx-lists.scss

```diff
@@ -301,3 +301,38 @@
   margin-left: var(--nx-spacing-6x);
   max-width: var(--nx-width-text-wrapping);
 }
+
+.nx-list--numbered {
+  padding: 0 0 0 var(--nx-spacing-12x);
+
+  .nx-list__item {
+    list-style: decimal outside;
+    padding: var(--nx-spacing-2x) 0 0 0;
+
+    .nx-list__text {
+      margin-left: 0;
+    }
+
+    .nx-list {
+      padding-left: var(--nx-spacing-12x);
+    }
+
+    .nx-list__item {
+      list-style: lower-alpha outside;
+      padding-top: var(--nx-spacing-2x);
```

Comment:
Also seems to be redundant

Reply:
> Also seems to be redundant

https://github.com/sonatype/sonatype-react-shared-components/pull/616/commits/f7098ffe2db3c2932434de0284879526230ccacf